editor/plugins/texture_editor_plugin.cpp

```diff
@@ -82,25 +113,51 @@ void TexturePreview::_update_texture_display_ratio() {
 	}
 }
 
-void TexturePreview::_update_metadata_label_text() {
-	const Ref<Texture2D> texture = texture_display->get_texture();
+static Image::Format get_texture_2d_format(const Ref<Texture2D> &p_texture) {
+	// Why is format not just available in the Texture class?
```

Comment:
Questions should be asked on chat or issues, not code :)

Reply:
I think I implicitely wanted it to come up, so... here it is. Either we could add that right now, or could be done in another PR.